### DISCUSSION TITLE
Testing DBC in Ubnuntu

### DISCUSSION THREAD

**Discussion by MichaelBMiner** [xR3b0rn/dbcppp#107] [General]
Hello @xR3b0rn,

I am making a script to read in an excel sheet and spit out a DBC. I am now trying to test my DBC but getting no output. 

This is a very short part of my DBC file. 
```
VERSION "1.0.0"

NS_ :
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: 

BO_ 408092926 PGN_21248_EVSE_1_Control_1: 8 EV
 SG_ EVSE1_Connector_Lock_Request : 0|4@1+ (1,0) [0|15] "" EVCC
 SG_ EVSE1_DC_Stage_Request : 4|4@1+ (1,0) [0|15] "" EVCC
 SG_ EV_Ready : 8|2@1+ (1,0) [0|3] "" EVCC
```

In two terminals I am running 
`candump any | dbcppp decode --bus=vcan0:file.dbc`
and
`cansend vcan0 185300FE#1122334455667788`

When I run this my candump gets no output. If I change my CANID to 1961 or 0x7A9 this works just fine. Is there a limit to the size of the CANID I am not aware of?

**Comment by xR3b0rn** [comment]
When sending CAN fd frames, you have to use `##`: `cansend vcan0 185300FE##1122334455667788`. You may receive an error when sending FD frames via `cansend`, use `ip link set vcan0 mtu 72` to fix it.
